tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -0,0 +1,228 @@
+
+    def set_configuration(self, config):
+        for key, value in config.items():
+            self.configuration[key] = value
+
+    def _create_benchmark_workload_file(self, node):
+        conf = self.render("omb_workload.yaml", **self.configuration)
+
+        self.logger.debug(
+            "Open Messaging Bechmark: Workload configuration: \n %s", conf)
+        node.account.create_file(OpenMessagingBenchmark.WORKLOAD_FILE, conf)
+
+    def _create_benchmark_driver_file(self, node):
+        rp_node = self.redpanda.nodes[0].account.hostname
+        conf = self.render("omb_driver.yaml", redpanda_node=rp_node)
+
+        self.logger.debug(
+            "Open Messaging Bechmark: Driver configuration: \n %s", conf)
+        node.account.create_file(OpenMessagingBenchmark.DRIVER_FILE, conf)
+
+    def _create_workers(self):
+        consumer_workers_amount = self.configuration[
+            "topics"] * self.configuration[
+                "subscriptions_per_topic"] * self.configuration[
+                    "consumer_per_subscription"]
+        producer_workers_amount = self.configuration[
+            "topics"] * self.configuration["producers_per_topic"]
+        workers_amount = consumer_workers_amount + producer_workers_amount
+        self.workers = OpenMessagingBenchmarkWorkers(
+            self._ctx, num_workers=workers_amount)
+        self.workers.start()
+
+    def start_node(self, node):
+        idx = self.idx(node)
+        self.logger.info("Open Messaging Bechmark: benchmark node - %d on %s",
+                         idx, node.account.hostname)
+
+        self.clean_node(node)
+
+        self._create_workers()
+
+        self._create_benchmark_workload_file(node)
+        self._create_benchmark_driver_file(node)
+
+        worker_nodes = self.workers.get_adresses()
+
+        self.logger.info(
+            f"Starting Open Messaging Bechmark with workers: {worker_nodes}")
+
+        start_cmd = f"cd {OpenMessagingBenchmark.OPENMESSAGING_DIR}; \
+                    bin/benchmark \
+                    --drivers {OpenMessagingBenchmark.DRIVER_FILE} \
+                    --workers {worker_nodes} \
+                    --output {OpenMessagingBenchmark.OUTPUT_FILE} \
+                    {OpenMessagingBenchmark.WORKLOAD_FILE} >> {OpenMessagingBenchmark.STDOUT_STDERR_CAPTURE} 2>&1 \
+                    & disown"
+
+        node.account.mkdirs(OpenMessagingBenchmark.PERSISTENT_ROOT)
+
+        with node.account.monitor_log(
+                OpenMessagingBenchmark.STDOUT_STDERR_CAPTURE) as monitor:
+            node.account.ssh(start_cmd)
+            monitor.wait_until(
+                "Starting warm-up traffic",
+                timeout_sec=20,
+                backoff_sec=4,
+                err_msg="Open Messaging Benchmark service didn't start")
+
+    def wait_node(self, node, timeout_sec):
+        process_pid = node.account.java_pids("benchmark")
+        if len(process_pid) == 0:
+            return True
```

Comment:
When the tests call `benchmark.wait(timeout_sec=becnhmark_time_min * 60)` the service implementation will loop over the nodes and invoke `wait_node`. However, if the process isn't present, `wait_node` return that everything is ok. But what if the process isn't present because it crashed?

How can we test that the process exited normally rather than crashed? Likewise below, should we check if the process exits cleanly?

I'm wondering it makes sense to have logic like:

if process does not exist:
   - check log file to see if it looks like the benchmark finished
else
   - stop the process

?